[DOCS-1608] Update Calico Enterprise for Windows docs for Operator + HPC installation (#1090)
tigera/docs: calico-enterprise/operations/monitor/prometheus/support.mdx

## Install options

### Use Prometheus operator managed by Tigera operator

You install the {{prodname}} Prometheus operator and CRDs during {{prodname}} installation. {{prodname}} metrics and alerts are available in Manager UI. You configure alerts through Prometheus AlertManager.

> If you want to specify your own Prometheus operator during installation for management by the Tigera operator, the require operator version must be **v0.40.0 or higher**. Because {{prodname}} creates AlertManager and Prometheus CRs in the `tigera-prometheus` namespace, all you need to do is verify that your Prometheus operator is configured to manage Prometheus and AlertManager instances in the `tigera-prometheus` namespace.

Reviewer: ```suggestion
If you want to specify your own Prometheus operator during installation for management by the Tigera operator, the required operator version must be **v0.40.0 or higher**. Because {{prodname}} creates AlertManager and Prometheus CRs in the `tigera-prometheus` namespace, all you need to do is verify that your Prometheus operator is configured to manage Prometheus and AlertManager instances in the `tigera-prometheus` namespace.
```

#### Prometheus on {{prodnameWindows}}

> By default, the Windows firewall blocks listening on ports. To have {{prodname}} manage the Prometheus metrics ports Windows firewall rules, enable the `windowsManageFirewallRules` setting in FelixConfiguration:

Reviewer: ```suggestion
By default, the Windows firewall blocks listening on ports. To allow {{prodname}} to manage the Prometheus metrics ports Windows firewall rules, enable the `windowsManageFirewallRules` setting in FelixConfiguration:
```

```bash
kubectl patch felixConfiguration default --type merge --patch '{"spec": {"windowsManageFirewallRules": "Enabled"}}'
```

> [See the FelixConfiguration reference for more details](../../../reference/resources/felixconfig.mdx). You can also add a Windows firewall rule allowing listening on the Prometheus ports instead of having {{prodname}} manage it.

Reviewer: ```suggestion
[See the FelixConfiguration reference for more details](../../../reference/resources/felixconfig.mdx). You can also add a Windows firewall rule that allows listening on the Prometheus ports instead of having {{prodname}} manage it.
```

### Bring your own Prometheus

> See here for more info on [bringing your own Prometheus](byo-prometheus.mdx) to {{prodname}}.

Reviewer: ```suggestion
For details, see [Bring your own Prometheus](byo-prometheus.mdx) to {{prodname}}.
```
